elasticms-admin/public/index.php

```diff
@@ -28,6 +28,5 @@
             | Request::HEADER_X_FORWARDED_PROTO
             | Request::HEADER_X_FORWARDED_PORT);
     }
-
```

Comment:
Not sure that PHPCS will agree with this?

Reply:
Will be rollbacked see my comment